packages/flutter/test/widgets/scrollable_test.dart

```diff
@@ -28,6 +28,13 @@ double getScrollOffset(WidgetTester tester) {
   return viewport.offset.pixels;
 }
 
+double getScrollVelocity(WidgetTester tester) {
+  final RenderViewport viewport = tester.renderObject(find.byType(Viewport));
+  final ScrollPosition position = viewport.offset;
+  // Access for test only.
```

Comment:
i'd prefer if the comment explaining the ignore was on the same line as the ignore, so it's clearer that they're linked

Reply:
From offline conversation with @xster: dart-lang/linter#706
+1'ed the bug as I'd also prefer `// ignore THING: reason`